spartan/terraform/deploy-release/main.tf

```diff
@@ -40,7 +40,7 @@ resource "helm_release" "aztec-gke-cluster" {
   upgrade_install  = true
 
   # base values file
-  values = [file("../../aztec-network/values/${var.VALUES_FILE}")]
+  values = [file("../../aztec-network/values/${var.VALUES_FILE}"), "${var.VALUES_INLINE}"]
```

Comment:
Hmmm, I haven't used this yet but my reasoning would be to use this after bootstrapping devnet to update the faucet with DevCoin.